amd/Dockerfile

```diff
@@ -0,0 +1,117 @@
+ARG BASE_IMAGE
```

Comment:
Could you please create a folder and then add the Dockefile file in it?  Should be `amd/ubi9-python-3.9/Dockerfile`
  
We this we follow the same folder pattern that we do for all the rest. For your ref,  check out  here: https://github.com/opendatahub-io/notebooks/tree/main/cuda

Reply:
Yes instead it would be c9s-python-3.9 we are holding off on ubi for right now